packages/e2e-test-utils/src/insertBlock.js

```diff
@@ -46,6 +46,7 @@ async function waitForInserterCloseAndContentFocus() {
 async function insertBlock(searchTerm) {
```

Comment:
Now need to remove the `Avoids using waitForInserterCloseAndContentFocus() [...]` comment above as it's no longer accurate.